lib/collective.lib.js

```diff
@@ -228,3 +228,8 @@ export const isSelfHostedAccount = c => c.isHost === true && c.type === 'COLLECT
 
 /* Returns true if the account is an individual. Works with GQLV1 (Collectives) & GQLV2 (Accounts) */
 export const isIndividualAccount = account => ['USER', 'INDIVIDUAL'].includes(account.type);
+
+/* Checks whether an account supports grants */
+export const accountSupportsGrants = (collectiveSettings, hostSettings) => {
+  return get(collectiveSettings, 'expenseTypes.hasGrant', get(hostSettings, 'disableGrantsByDefault', false));
```

Comment:
I think `get` only returns the third param if it gets undefined. So this approach will prevent disabling grants (opt-out) for a specific collective. 

Also, we don't want to return true if the flag to disable grants is true.

Reply:
If `disableGrantsByDefault` is true, `accountSupportsGrants` should return false.

```suggestion
  return get(collectiveSettings, 'expenseTypes.hasGrant', !get(hostSettings, 'disableGrantsByDefault', false));
```

Reply:
Ops, a mistake on my part. I've corrected this. 👍🏽